### ISSUE TITLE
Line numbers not separated from code

### ISSUE THREAD

**Issue by pvcraven** [sphinx-doc/sphinx#9363]
**Describe the bug**
Starting with Sphinx 4.0.0 copying a block of code with `:linenos:` includes the line numbers along with the code when copied.

**To Reproduce**
Create an RST that references an external file, something like:
```
.. literalinclude:: array_backed_grid.py
    :caption: array_backed_grid.py
    :linenos:
```

**Expected behavior**
The ability to copy code without line numbers. This worked fine with Sphinx 3.5.7 and before.Example:

<img width="393" alt="s2" src="https://user-images.githubusercontent.com/1463116/122603680-55237600-d03a-11eb-95d7-26fbbd0da906.png">

Starting with 4.0.0 we get:

<img width="426" alt="s1" src="https://user-images.githubusercontent.com/1463116/122603681-55237600-d03a-11eb-992d-81c5f1c63c20.png">

Sphinx copy button extension includes line numbers as well.

**Environment info**
- OS: Windows 10
- Python version: 3.9.5
- Sphinx version: 4.0.0 to 4.0.2

**Comment by mgeier** [comment]
What HTML theme are you using?

I've fixed this for the `basic` theme in #8320.

You should probably report this to the issue tracker of the appropriate HTML theme.

As a temporary work-around, you can try to use `html_codeblock_linenos_style = 'table'`, but this option will be removed at some point.

**Comment by pvcraven** [comment]
I'm using the sphinx-rtd-theme. Didn't think this would  be controlled by the theme. Thanks for the info and work-around.
